CoinDock-api/app/Http/Controllers/V1/Auth/UserController.php

```diff
@@ -7,13 +7,46 @@
 use App\Http\Requests\V1\SignupRequest;
 use Illuminate\Http\Request;
 use App\Models\V1\User;
+use App\Models\V1\SignUp;
 use Illuminate\Support\Facades\Auth;
 
 class UserController extends Controller
 {
     /**
      * SignupRequest
      */
+    public function signUpInfo(User $user)
```

Comment:
Should be in its own controller SignupController.
Return user specific data using the show() method

Reply:
done